src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,153 @@
+        String sqlQuery = String.format("DELETE FROM \"%1$s\" WHERE %2$s = ? ", tableName, columnName);
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sqlQuery)) {
+            preparedStatement.setString(1, title);
+            preparedStatement.executeUpdate();
+        }
+    }
+
+    private void updateDataByTitle(String oldData, String newData, String tableName, String columnName) throws SQLException {
+        String sqlQuery = String.format("UPDATE \"%3$s\" SET %4$s = \'%2$s\' WHERE %4$s = \'%1s\';", oldData, newData, tableName, columnName);
+        try (Statement statement = connection.createStatement()) {
+            statement.executeUpdate(sqlQuery);
+        }
+    }
+
+    private void showExistingRecordsOfTable(String tableName) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            if (tableName != null) {
+                String sqlQuery = String.format("SELECT * FROM \"%s\"", tableName);
+                ResultSet resultSet = statement.executeQuery(sqlQuery);
+                int columnCount = resultSet.getMetaData().getColumnCount();
+
+                StringBuilder tableData = new StringBuilder();
+                while (resultSet.next()) {
+                    for (int i = 1; i <= columnCount; i++) {
+                        tableData.append(resultSet.getString(i)).append(" | ");
+                    }
+                    tableData.replace(tableData.length() - 3, tableData.length(), LINE_SEPARATOR);
+                }
+                System.out.println(tableData.toString());
+            }
+        }
+
     }
+
+    private void addUser(String userName, String userPassword) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            if (userName != null && userPassword != null) {
+                // Why didn't suit here such expression "INSERT INTO public." + "\"" + TABLE_NAME + "\"" + "(password, name) "
+                //                                      + "VALUES (" + userPassword + ", " + userName + ");";
+                String sqlQuery = String.format("INSERT INTO \"" + TABLE_NAME + "\"(name, password) VALUES('%s','%s') ", userName, userPassword);
+                statement.executeUpdate(sqlQuery);
+            } else {
+                System.out.println("Wrong input");
+            }
+        }
+    }
+
+    private void dropTableWithName(String user) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String sqlQuery = "DROP TABLE IF EXISTS \"" + user + "\" CASCADE ;";
+            statement.executeUpdate(sqlQuery);
+        }
+    }
+
+    private void createTable(String tableName) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String sqlQuery = "CREATE TABLE public.\"" + tableName + "\"" + LINE_SEPARATOR
```

Comment:
Query doesn't need any LINE_SEPARATORs to be executed. Whole query could be in one line. Gether it with String.format